OpenTabletDriver.Plugin/Output/RelativeOutputMode.cs

```diff
@@ -1,126 +1,65 @@
+        public RelativeOutputMode()
         {
-            set
+            Config.PropertyChanged += (_, args) =>
             {
-                this.sensitivity = value;
-                UpdateTransformMatrix();
-            }
-            get => this.sensitivity;
+                if (!skipPropertyCheck)
+                {
+                    skipPropertyCheck = true;
+                    switch (args.PropertyName)
+                    {
+                        case "AreaLimiting":
+                            Config.AreaLimiting = false;
+                            break;
+                        case "AreaClipping":
+                            Config.AreaClipping = false;
+                            break;
+                    }
+                    skipPropertyCheck = false;
+                }
+            };
         }
 
-        private float rotation;
-        public float Rotation
-        {
-            set
-            {
-                this.rotation = value;
-                UpdateTransformMatrix();
-            }
-            get => this.rotation;
-        }
+        private Vector2? lastPos;
+        private HPETDeltaStopwatch stopwatch = new HPETDeltaStopwatch(true);
+        private double reportMsAvg = 10;
+        private bool skipReport, skipPropertyCheck;
 
-        private void UpdateTransformMatrix()
+        protected override void UpdateTransformMatrix()
         {
             this.skipReport = true; // Prevents cursor from jumping on sensitivity change
-
-            this.transformationMatrix = Matrix3x2.CreateRotation(
-                (float)(-Rotation * System.Math.PI / 180));
-
-            this.transformationMatrix *= Matrix3x2.CreateScale(
-                sensitivity.X * ((Tablet?.Digitizer?.Width / Tablet?.Digitizer?.MaxX) ?? 0.01f),
-                sensitivity.Y * ((Tablet?.Digitizer?.Height / Tablet?.Digitizer?.MaxY) ?? 0.01f));
+            base.UpdateTransformMatrix();
         }
 
-        public TimeSpan ResetTime { set; get; }
-
-        public virtual void Read(IDeviceReport report)
+        protected override Vector2? Transpose(ITabletReport report)
         {
-            if (report is ITabletReport tabletReport)
-            {
-                if (Tablet.Digitizer.ActiveReportID.IsInRange(tabletReport.ReportID))
-                {
-                    if (Pointer is IVirtualTablet pressureHandler)
-                        pressureHandler.SetPressure((float)tabletReport.Pressure / (float)Tablet.Digitizer.MaxPressure);
+            var pos = base.Transpose(report);
+            var deltaTime = stopwatch.Restart().TotalMilliseconds;
+            reportMsAvg += (deltaTime - reportMsAvg) * 0.1;
+            Vector2? delta;
 
-                    if (Transpose(tabletReport) is Vector2 position)
-                        Pointer.Translate(position);
-                }
+            if (lastPos.HasValue && pos.HasValue && !skipReport && AsyncFilterHandler.WithinDelayTolerance(deltaTime, reportMsAvg))
```

Comment:
In this case of just checking nullable's value lazy eval will only hurt performance by unnecessary branching
```suggestion
            if (lastPos.HasValue & pos.HasValue & !skipReport & AsyncFilterHandler.WithinDelayTolerance(deltaTime, reportMsAvg))
```